package.json

```diff
@@ -60,5 +60,8 @@
   "packageManager": "pnpm@8.14.0",
   "engines": {
     "node": "20.x"
+  },
+  "dependencies": {
+    "turbo": "^1.11.3"
```

Comment:
I'm *slightly* opposed to pinning the version here as opposed to pulling the latest version/canary at time of run. If we do go ahead with pinning a `turbo` version, then it should probably be canary so we're on the bleeding edge.

Reply:
Oh, oops, this was an accident. Thanks for noticing.